CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [3.3.0-cr2] - 2018-09-25
```

Comment:
Isn't it rc2?

Reply:
:man_facepalming: 
Copy-pasted from rc1 which is wrong.

Reply:
let's keep 'rc' for 3.3.0 at least so we don't have to change the existing rc1 tag.